test/taxCalculator.test.js

```diff
@@ -114,14 +148,16 @@ describe('Calculate taxes', function () {
                 { name: 'Tax level 7', rate: 0.35, amount: 1_625_400 }],
             totalTax: 19775400
         };
-        assert.deepStrictEqual(actualTaxes, expectedTaxes);
+        await calculateTaxes(84_644_000).then(function (taxes) {
+            assert.deepStrictEqual(taxes, expectedTaxes)
+        })
     })
 
-    test('Total tax is equal total of tax per rate ', function () {
+    test('Total tax is equal total of tax per rate ', async function () {
         //tax.totalTax = tax.totalTax.plus(taxPaidForEachRate);
 
-        const actualTaxes = calculateTaxes(29_000_000);
-        assert.deepStrictEqual(actualTaxes.totalTax, 4_150_000);
+        await calculateTaxes(29_000_000).then(function (taxes) {
+            assert.deepStrictEqual(taxes.totalTax, 4_150_000)
```

Comment:
should calc sum of paid tax per rates from taxes And then assert that sum with taxes.totalTax